src/Nlog.RabbitMQ.Target/RabbitMQTarget.cs

```diff
@@ -351,7 +365,7 @@ private string GetMessageString(LogEventInfo logEvent)
 					var jsonSerializer = JsonSerializer;
 					lock (jsonSerializer)
 					{
-						return MessageFormatter.GetMessageInner(jsonSerializer, message, messageSource, logEvent, this.Fields);
+						return MessageFormatter.GetMessageInner(jsonSerializer, AddGdc, AddNdlc, AddMdlc, message, messageSource, logEvent, this.Fields);

```

Comment:
Maybe collect the properties in the RabbitMQTarget and instead just pass a Dictionary to the MessageFormatter